bndtools.core.test/build.gradle

```diff
@@ -0,0 +1,37 @@
+import aQute.bnd.gradle.TestOSGi
+import org.apache.tools.ant.taskdefs.condition.Os
+
+task testosgi(type: TestOSGi) {
+	[
+		'biz.aQute.bndlib',
+		'biz.aQute.repository',
+		'biz.aQute.resolve',
+		'biz.aQute.tester.junit-platform',
+		'bndtools.api',
+		'bndtools.builder',
+		'bndtools.core',
+		'biz.aQute.tester.junit-platform',
+		'org.bndtools.headless.build.manager',
+		'org.bndtools.headless.build.plugin.ant',
+		'org.bndtools.headless.build.plugin.gradle',
+		'org.bndtools.templates.template',
+		'org.bndtools.templating.gitrepo',
+		'org.bndtools.templating',
+		'org.bndtools.versioncontrol.ignores.manager',
+		'org.bndtools.versioncontrol.ignores.plugin.git'
+	].each { dependsOn ":$it:jar" }
+   description "Bndtools Core Integration tests"
+   group 'test'
+   inputs.files jar
+   if (Os.isFamily(Os.FAMILY_WINDOWS)) {
```

Comment:
Is this the gradle way to do this?

Reply:
It's the best i found- if you know a better way,  let me know!

Reply:
Apparently `Os.isFamily(Os.FAMILY_UNIX)` returns true on macOS. Which means this if/else construct is broken.

We do this elsewhere in the build:

https://github.com/bndtools/bnd/blob/35661c2dd92f56993c78cf6c984323af0f41d42f/maven/build.gradle#L40

Reply:
I'm not sure what "os.name" is going to throw at me on the various OSes. Using `Os.isFamily()` is voted the best solution on [this](https://stackoverflow.com/questions/11235614/how-to-detect-the-current-os-from-gradle) SO thread, though the answer is old. I think it's a bit of an oversight that Gradle doesn't have any built-in support for this.

I have reorded the if/else so that Mac is tested first and Linux is only used if it's not Mac. I think it's more compact. If you really prefer though, I can hand-craft something based on inspection of `os.name` (I just don't have a good way to verify it as I don't have access to a Mac to test it on).